### ISSUE TITLE
[BUG] Error in tutorial notebook 02_Catalog_Queries_with_TAP

### ISSUE THREAD

**Issue by Kabelo-T** [rubin-dp0/Support#35]
**Describe the bug**

Queries take much longer than the tutorial notebook suggests.

**To Reproduce**
1. Go to the tutorial notebooks on the RSP instance on the IDF under /notebooks
2. Click on 02_Catalog_Queries_with_TAP
3. Scroll down to section 3.2. Cone search specifying the maximum number of records

**Expected behavior**
Small error in the notebook, in every cell that says the queries should take 2-10s they actually usually take close to 4mins.

**Screenshots**
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/100869333/183783920-aa3f37e3-3a7c-4058-b156-e857e4bbd7bf.png">

**URL**
https://data.lsst.cloud/nb/user/kabelo-t/lab/tree/notebooks/tutorial-notebooks/02_Catalog_Queries_with_TAP.ipynb

**Desktop (please complete the following information):**
 - OS: [MacOS 12.5]
 - Browser [Safari]

**Comment by MelissaGraham** [comment]
My apologies @Kabelo-T, this issue completely escaped my notice.

For staff, it is being addressed with jira.lsstcorp.org/browse/PREOPS-1363.

**Comment by MelissaGraham** [comment]
OK, I did some tests, and the cells that can take just a few seconds do sometimes take more time, but always <2 minutes. Changed all text to say "This query usually takes <2 minutes" to avoid future confusion. But there's nothing wrong with the queries or anything.

Changes have been pushed to main and so will appear to users in the next release to prod, by Patch Thu Sep 15 if not before.

Closing issue.
